pkg/notes/options/options.go

```diff
@@ -83,6 +91,7 @@ func New() *Options {
 // ValidateAndFinish checks if the options are set in a consistent way and
 // adapts them if necessary. It returns an error if options are set to invalid
 // values.
+// nolint:gocyclo
```

Comment:
This should not be needed any more with the merge of https://github.com/kubernetes/release/pull/1147

Reply:
Done.